fullstack-cert/python-projects/booleans-conditonals-lab/user-stories.md

```diff
@@ -0,0 +1,21 @@
+### Commuting Decision Logic (Script)
+**Objective:** Fulfil the user stories below and get all the tests to pass to complete the lab.
+1. You should create the following variables:
+   * `distance_mi` (a number representing the distance to travel)
+   * `is_raining` (a boolean)
+   * `has_bike` (a boolean)
+   * `has_car` (a boolean)
+   * `has_ride_share_app` (a boolean)
```

Comment:
these four variables should also have a description of what they represent with a sentence, let's not rely on their name